utils/src/main/java/software/amazon/awssdk/utils/Validate.java

```diff
@@ -47,7 +47,7 @@
  * @see java.lang.String#format(String, Object...)
  */
 @ReviewBeforeRelease("Remove the methods we don't end up using (and software.amazon.awssdk.core.util.ValidationUtils).")
-@SdkInternalApi
+@SdkProtectedApi
```

Comment:
Public

Reply:
Before GA, sure. It's not ready to be public right now, though.